core/src/toga/window.py

```diff
@@ -267,7 +267,7 @@ def resizable(self) -> bool:
 
     @property
     def _default_title(self) -> str:
-        return "Toga"
+        return toga.App.app.formal_name if toga.App.app else "Toga"
```

Comment:
It is in practice; but there's a unit test that validates this case. If you create an instance of toga.Window before you create an app, `App.app` won't exist yet. You then get an error that reports the actual problem (there's no app), rather than a "no such attribute" exception.

Reply:
As discussed in person: If we re-order the check for App existing, the need for the special case disappears. I've made this change.